<?xml version="1.0"?>

<launch>

  <node
        name = "potential_field_controller"
        pkg = "move_package"
        type = "pot_spread.py"    
        output = "screen"
        args = "none"
    >
  </node>

  <node
        name = "uav1potential_field_controller"
        pkg = "move_package"
        type = "pot_spread.py"    
        output = "screen"
        args = "uav1"
    >
  </node>

  <node
        name = "uav2potential_field_controller"
        pkg = "move_package"
        type = "pot_spread.py"    
        output = "screen"
        args = "uav2"
    >
  </node>

</launch>